lib/models/LifecycleConfiguration.js

```diff
@@ -201,14 +231,20 @@ class LifecycleConfiguration {
                 customizeDescription('Rule xml does not include Status');
             return ruleObj;
         }
-        const subFilter = rule.Filter ? rule.Filter[0] : rule.Prefix;
-
         const id = this._parseID(rule.ID);
         const status = this._parseStatus(rule.Status[0]);
-        const filter = this._parseFilter(subFilter);
         const actions = this._parseAction(rule);
-
-        const rulePropArray = [id, status, filter, actions];
+        const rulePropArray = [id, status, actions];
+        let prefix = {};
```

Comment:
`prefix` and `filter` should be local to the blocks below where they're used since they're not accessed outside.

Reply:
Thanks, I think that was left behind from a refactor that didn't get updated properly. Changing this now.